soccer_pycontrol/src/soccer_pycontrol/test_walking.py

```diff
@@ -35,12 +38,29 @@
 
     rospy.Time = MagicMock()
     joint_state = MagicMock()
-    joint_state.position = [0.0] * 18
+    joint_state.position = [0.0] * 20
     rospy.wait_for_message = MagicMock(return_value=joint_state)
     rospy.loginfo_throttle = lambda a, b: None
-    rospy.get_param = lambda a, b: b
+
```

Comment:
put this function and robot_model into TestWalking

Reply:
Instea used parameterized unittests 

https://stackoverflow.com/questions/32899/how-do-you-generate-dynamic-parameterized-unit-tests-in-python

So you can run the same test for all 3 robots